src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rezult = {};
```

Comment:
Consider using a more descriptive variable name instead of 'rezult'. For example, 'invertedRobot'.